src/components/ToggleButton.tsx

```diff
@@ -0,0 +1,124 @@
+import React, {ReactNode} from "react";
+import {Animated, StyleSheet, TouchableWithoutFeedback, View} from "react-native";
+import Colors from "../constants/Colors";
+
+export interface ToggleButtonProps {
+    backgroundColor?: string;
+    isToggled?: boolean;
+    onToggle?: (isToggled: boolean) => void;
+    textColorIn?: string;
+    textColorOut?: string;
+    title: string;
+}
+
+interface ToggleButtonState {
+    isToggled: boolean;
+}
+
+const ANIMATION_TIME_MS = 75;
+
+export class ToggleButton extends React.Component<ToggleButtonProps, ToggleButtonState> {
+
+    public get isToggled(): boolean {
+        return this.props.isToggled;
```

Comment:
Shouldn't this be
```suggestion
        return this.state.isToggled;
```
@richardkriesman? :p

Reply:
Wait, this has apparently already been fixed. Cool